mod/cp_notifications/languages/en.php

```diff
@@ -20,7 +20,7 @@
 	'minor_save:description' => "Posting new content will send out notifications to those who are subscribed. As the group owner or operator, you can decide not to send out notifications about the new content you post in the group. To do so, select the option “Do not send a notification” below.",
 	'minor_save:checkbox_label' => " Do not send a notification",
 
-	'cp_notifications:name' => "Notification Email",
+	'cp_notifications:name' => "Notification settings",
```

Comment:
I think we may have a language string that is translated for notification settings. https://github.com/gctools-outilsgc/gcconnex/blob/master/mod/wet4/languages/en.php#L742
try `notifications:usersettings`

Reply:
@piet0024 Part of me renaming that from email to settings was Ilia found that the title of the notification settings page was incorrect in English. It was using that language string, so I fixed the English version only. There is a ticket created for it #2446